Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -91,32 +101,75 @@ void HiresTexture::Update()
 
   const std::string& game_id = SConfig::GetInstance().GetGameID();
   const std::string texture_directory = GetTextureDirectory(game_id);
-  std::vector<std::string> extensions{
-      ".png", ".bmp", ".tga", ".dds",
-      ".jpg"  // Why not? Could be useful for large photo-like textures
-  };
+  std::vector<std::string> extensions{".dds", ".png", ".bmp",  ".tga",  ".tgb",
+                                      ".jpg", ".jpe", ".jpeg", ".jfif", ".hdr"};
 
-  std::vector<std::string> filenames =
+  std::vector<std::string> filepaths_found =
       Common::DoFileSearch({texture_directory}, extensions, /*recursive*/ true);
 
   const std::string code = game_id + "_";
 
-  for (auto& rFilename : filenames)
+  FileType_Counter_DDS_Amount = 0;
+  FileType_Counter_PNG_Amount = 0;
+  FileType_Counter_BMP_Amount = 0;
+  FileType_Counter_TGA_Amount = 0;  // Common for TGA and TGB vatiants
+  FileType_Counter_JPG_Amount = 0;  // Common for JPG, JPE, JPEG and JFIF variants
+  FileType_Counter_HDR_Amount = 0;
+  FileType_Counter_Total_Amount = 0;
+
+  for (auto& full_filepath : filepaths_found)
   {
-    std::string FileName;
-    SplitPath(rFilename, nullptr, &FileName, nullptr);
+    std::string current_filename;
+    std::string current_filetype;
```

Comment:
since when is that two words, i've never came across that, what about "base_filename" there's a thousand of those. I don't mind doing it this case but it doesn't make sense to me looking at the overall thing, that's why I'm more keen on making big PRs where a lot of stuff like this can be sweeped up in one go, much less frustrating to code.